WalletWasabi.Gui/Controls/MultiTextBox.cs

```diff
@@ -193,7 +193,7 @@ private async Task OnClickedAsync(EventPattern<object> eventPattern)
 			{
 				var eventArgs = eventPattern?.EventArgs as PointerPressedEventArgs;
 				var uiConfig = Locator.Current.GetService<Global>().UiConfig;
-				if (uiConfig.Autocopy is true && eventArgs?.GetCurrentPoint(this).Properties.IsLeftButtonPressed == true)
+				if (uiConfig.Autocopy && eventArgs?.GetCurrentPoint(this).Properties.IsLeftButtonPressed is true)
```

Comment:
remove the `is true`

Reply:
Here it shouldn't be removed.
`eventArgs?.GetCurrentPoint(this).Properties.IsLeftButtonPressed` returns `bool?`